src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -569,4 +569,23 @@ public String getThriftProtocolClass() {
     public String getMetricsCollectorClass() {
         return mProperties.getString("secor.monitoring.metrics.collector.class");
     }
+    
+    public Map<String, String> getPropertyMapForPrefix(String prefix) {
```

Comment:
Put a javadoc to give an example of the content in the return map.

And for your config, if people specified secor.orc.message.schema.*=abc, does your map return key="*", value="abc"?